virtool/startup.py

```diff
@@ -359,3 +362,24 @@ async def init_version(app: typing.Union[dict, aiohttp.web.Application]):
     logger.info(f"Mode: {app['mode']}")
 
     app["version"] = version
+
+
+async def init_tasks(app: aiohttp.web.Application):
+    db = app["db"]
+    settings = app["settings"]
+    logger.info("Checking subtraction files")
+
+    subtraction_without_file = await virtool.subtractions.utils.check_subtraction_file(app)
+    scheduler = get_scheduler_from_app(app)
+
+    if subtraction_without_file:
+        for subtraction_id in subtraction_without_file:
```

Comment:
I want a single task to go through each subtraction sequentially. Start a separate task for each subtraction could cause resource issues.